sdks/python/README.md

```diff
@@ -339,89 +340,91 @@ text file from [Google Cloud Storage](https://cloud.google.com/storage/).
 
 ```python
 import re
-import google.cloud.dataflow as df
-p = df.Pipeline('DirectPipelineRunner')
+import apache_beam as beam
+p = beam.Pipeline('DirectPipelineRunner')
 (p
- | df.Read('read',
-           df.io.TextFileSource(
-           'gs://dataflow-samples/shakespeare/kinglear.txt'))
- | df.FlatMap('split', lambda x: re.findall(r'\w+', x))
- | df.combiners.Count.PerElement('count words')
- | df.Write('write', df.io.TextFileSink('./results')))
+ | 'read' >> beam.Read(
+        beam.io.TextFileSource('gs://dataflow-samples/shakespeare/kinglear.txt')
+    )
+ | 'split' >> beam.FlatMap(lambda x: re.findall(r'\w+', x))
+ | 'count words' >> beam.combiners.Count.PerElement()
+ | 'save' >> beam.Write(beam.io.TextFileSink('./word_count')))
 p.run()
 ```
 
 ### Counting words with GroupByKey
 
-Here we use `GroupByKey` to count the words.
-This is a somewhat forced example of `GroupByKey`; normally one would use
-the transform `df.combiners.Count.PerElement`, as in the previous example.
-The example also shows the use of a wild-card in specifying the text file
-source.
-
+This is a somewhat forced example of `GroupByKey` to count the words.
+Normally, one would use the transform `beam.combiners.Count.PerElement`,
+as in the previous example. A wildcard is used to specify the text
```

Comment:
This is fine either way.

I mean keeping the whole sentence "The example also shows the use of a wild-card in specifying the text file source." would be better than just "A wildcard is used to specify the text file source"